src/lib/utils/router.js

```diff
@@ -113,7 +114,7 @@ export function listen(handler) {
       // If user has a preferred language, use it in the request.
       const lang = store.getState().userPreferredLanguage;
       if (lang && lang !== language.defaultLanguage) {
-        url = '/' + lang + url;
+        url = path.join('/', lang, url);
```

Comment:
ah I overlooked the fact that `path` isn't available in the browser! 🤦‍♂️

I think `path-browserify` adds 53kb to the bundle when it's unpacked so it's probably not worth the size. I think we can just use what you had before: `url = '/' + lang + url;`

Reply:
ok!